### ISSUE TITLE
Rich Text Editor's color picker now showing black color as default in dark color scheme.

### ISSUE THREAD

**Issue by fantasyz** [mantinedev/mantine#4559]
### What package has an issue

@mantine/tiptap

### Describe the bug

In dark theme, the default text color is not black but the button is showing black when cursor stopped at default color text. 

![sample](https://github.com/mantinedev/mantine/assets/3223242/aa641405-9d4b-4439-b110-f0f39bc5b4ac)

### What version of @mantine/hooks page do you have in package.json?

"@mantine/tiptap": "^6.0.16", "@tiptap/extension-color": "^2.0.4"

### If possible, please include a link to a codesandbox with the reproduced problem

It also happen on your demo page:
https://mantine.dev/others/tiptap/

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by hellolol2016** [comment]
It seems like I'll just need to brute force the color to match when it's in dark mode. I'll take a crack at it for now.

**Comment by hellolol2016** [comment]
I think I found the issue on the file ColorPickerControl.tsx, but I'm not sure how to change it so that it's dynamic for the different themes. 

It seems that the editor.getAttributes doesn't actually successfully pull the text color so it defaults to #000. I'm not really sure how to proceed from here. Any tips would be appreciated.

![image](https://github.com/mantinedev/mantine/assets/66754085/b22fd6ed-c4ad-4258-aff4-1de73ef7b399)

**Comment by fantasyz** [comment]
It's from your website sample.

```
import { Text } from '@mantine/core';

function Demo() {
  return (
    <Text
      sx={(theme) => ({
        // subscribe to color scheme changes
        backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.blue[0],

        // or use any other static values from theme
        color: theme.colors.gray[1],
        fontSize: theme.fontSizes.sm,
        fontWeight: 500,
      })}
    >
      My custom text
    </Text>
  );
}
```

Maybe you can just directly use theme.colors.xxx[?] based on theme and the actual index used for the default text color?

```
const theme = useMantineTheme();
const currentColor = theme.colorScheme === 'dark' ? theme.colors.dark[1] : theme.colors.blue[1],
```

**Comment by hellolol2016** [comment]
I see, I'll try implementing that and submitting a pull request. Any ideas on how I can test this on my machine?

**Comment by fantasyz** [comment]
I don't know. I have no experience dealing with lib source. I just started to learn react 2 months ago.

**Comment by rtivital** [comment]
Fixed in 6.0.20
